gradle/libs.versions.toml

```
@@ -66,6 +65,7 @@ build-japicmp-plugin = "me.champeau.gradle:japicmp-gradle-plugin:0.4.2"
 build-japicmp = { module = "com.github.siom79.japicmp:japicmp", version.require = "0.15.+" }
 build-nullaway = "net.ltgt.gradle:gradle-nullaway-plugin:1.6.0"
 build-spotless = { module = "com.diffplug.spotless:spotless-plugin-gradle", version.ref = "spotless" }
+build-shadow = "com.github.johnrengelman:shadow:8.1.1"
```

```suggestion
build-shadow = { module = "com.github.johnrengelman:shadow", version = "8.1.1" }
```

i'd like to start moving things towards this more consistent syntax (the rest of this catalog should be updated, but you don't need to do that in this PR)